python/py-selenium/Portfile

```diff
@@ -1,30 +1,33 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-PortGroup       python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-name            py-selenium
-set real_name   selenium
-version         3.141.0
-python.versions 27
-license         Apache-2
-maintainers     {dstrubbe @dstrubbe} openmaintainer
-platforms       darwin
-description     Python language binding for Selenium Remote Control
+name                py-selenium
+set real_name       selenium
```

Comment:
there is no need for this variable, so please remove and where it's used below in the Portfile use ```python.rootname``` instead

Reply:
Done thx.